### PR TITLE
feat(searxng): use only 2 instances instead of 20+ instances

### PR DESCRIPTION
**Pull request by somerand0mcat**
## 🔧 What does this fix? 🔧
as said in the title

## 🗒 Checklist 🗒

- [X] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [X] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

**Comment by ryanccn** [comment]
One wonders if it would be better to just have only one or two default instances for simplicity

**Comment by somerand0mcat** [comment]
> One wonders if it would be better to just have only one or two default instances for simplicity

what if the user that uses this style is using another instance

**Comment by isabelroses** [comment]
> what if the user that uses this style is using another instance that is not these two

Just add it. We have a little tutorial on how to do it on the readme.

File: styles/searxng/catppuccin.user.css [hunk 1]
```diff
@@ -17,120 +17,9 @@
...
-domain("sx.catgirl.cloud"),
-url-prefix("https://timdor.noip.me/searx/"),
-url-prefix("https://trydex.tk/searxng/"),
-domain("www.gruble.de"),
-domain("www.webrats.xyz"),
-domain("xcxc.ml"),
-domain("xo.wtf") {
+@-moz-document
+domain("https://search.im-in.space/"), 
+domain("https://search.projectsegfau.lt/") {
```
Comment (uncenter) [inline styles/searxng/catppuccin.user.css:22]: Possibly stupid question but why these two specifically?

Comment (somerand0mcat) [reply styles/searxng/catppuccin.user.css:22]: > Possibly stupid question but why these two specifically?

i just googled the instances list and picked these two

Comment (somerand0mcat) [reply styles/searxng/catppuccin.user.css:22]: also they're pretty stable too

Comment (uncenter) [reply styles/searxng/catppuccin.user.css:22]: These two weren't even on the original list though?

Comment (uncenter) [reply styles/searxng/catppuccin.user.css:22]: Do you use SearXNG?

Comment (somerand0mcat) [reply styles/searxng/catppuccin.user.css:22]: a bit, yeah

Comment (uncenter) [reply styles/searxng/catppuccin.user.css:22]: Maybe just take the top three or something? Also leave a comment above the `@-moz-document` detailing where these are from would be great - `/* Domains picked from https://searx.space/. */

Comment (somerand0mcat) [reply styles/searxng/catppuccin.user.css:22]: that sounds like a great idea

Comment (somerand0mcat) [reply styles/searxng/catppuccin.user.css]: @uncenter i did all of your suggestions

**Review by uncenter — APPROVED** [review]

**Review by ryanccn — APPROVED** [review]

**Pull request merged**